charts/linkerd-control-plane/Chart.yaml

```diff
@@ -16,8 +16,7 @@ dependencies:
 - name: partials
   version: 0.1.0
   repository: file://../partials
-# this version will be updated by the CI before publishing the Helm tarball
-version: 0.1.0
+version: 1.0.0-edge
```

Comment:
So do we begin updating this now with the year.month.edge-number now? If not, when is this bumped?

Reply:
Unlike `appVersion`, this has to follow semver's `major.minor.patch[-pre-release identifier]`. I proposed in #7405 the following bumping criteria:

- patch: bump for minor changes
- minor: bump for new additions in templates or to values.yaml
- major: bump for backwards-incompatible changes made to values.yaml

Reply:
Sounds good. So lets say `edge-21.12.3` is the first edge with these changes.

The GitHub release and `linkerd version` output will still be `edge-21.12.3`, but the Helm version for linkerd-control-plane would be `1.0.1-edge`?

Reply:
Correct. For `linkerd-control-plane`, helm will expose both the chart and the app versions:
```
$ helm ls -A
NAME                    NAMESPACE       REVISION        UPDATED                                 STATUS          CHART                                   APP VERSION 
linkerd-crds            linkerd         1               2021-12-10 11:47:32.885183008 -0500 -05 deployed        linkerd-crds-1.0.0-edge                             
linkerd-control-plane   linkerd         1               2021-12-10 11:47:59.548941284 -0500 -05 deployed        linkerd-control-plane-1.0.0-edge        edge-21.12.2
```